internal/prompts/app_select_test.go

```diff
@@ -2455,2328 +1313,192 @@ func TestPrompt_AppSelectPrompt_FlatAppSelectPrompt(t *testing.T) {
-		}, {
-			"valid flags: --app local --team team1",
-			ShowLocalOnly,
-			ShowInstalledAppsOnly,
-			"local",
-			team1TeamDomain,
-			nil,
-			SelectedApp{Auth: fakeAuthsByTeamDomain[team1TeamDomain], App: appTeam1Local},
-		}, {
-			"valid flags: --app <local_id>",
-			ShowLocalOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE02",
-			"",
-			nil,
-			SelectedApp{Auth: fakeAuthsByTeamDomain[team1TeamDomain], App: appTeam1Local},
-		}, {
-			"valid flags: --app <local_id> --team team1",
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE01",
-			team1TeamDomain,
-			nil,
-			SelectedApp{Auth: fakeAuthsByTeamDomain[team1TeamDomain], App: appTeam1Hosted},
-		}, {
-			"invalid flags: --app <team1_local_app_id> --team team2",
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE01",
-			team2TeamDomain,
-			slackerror.New(slackerror.ErrAppNotFound),
-			SelectedApp{},
-		}, {
-			"invalid flags: --app <unknown_app_id>",
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE04",
-			"",
-			slackerror.New(slackerror.ErrAppNotFound),
-			SelectedApp{},
-		}, {
-			"invalid flags: --app <team1_local_app_id> --team team2",
-			ShowLocalOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE02",
-			team2TeamDomain,
-			slackerror.New(slackerror.ErrAppNotFound),
-			SelectedApp{},
-		}, {
-			"invalid flags for local only: --app <deploy_app_id>",
-			ShowLocalOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE01",
-			"",
-			slackerror.New(slackerror.ErrDeployedAppNotSupported),
-			SelectedApp{},
-		}, {
-			"invalid flags for deploy only: --app <local_app_id>",
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE02",
-			"",
-			slackerror.New(slackerror.ErrLocalAppNotSupported),
-			SelectedApp{},
-		}, {
-			"invalid flags for local only: --app <team1_deploy_app_id> --team team1",
-			ShowLocalOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE01",
-			team1TeamDomain,
-			slackerror.New(slackerror.ErrDeployedAppNotSupported),
-			SelectedApp{},
-		}, {
-			"invalid flags for deploy only: --app <team1_local_app_id> --team team1",
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-			"A1EXAMPLE02",
-			team1TeamDomain,
-			slackerror.New(slackerror.ErrLocalAppNotSupported),
-			SelectedApp{},
-		},
-	}
-
-	for _, test := range tests {
-		clients.Config.AppFlag = test.appFlag
-		clients.Config.TeamFlag = test.teamFlag
-		actualSelected, err := TeamAppSelectPrompt(ctx, clients, test.env, test.status)
-
-		if test.err == nil {
-			require.NoError(t, err)
-
-			_, err := clients.AppClient().GetDeployed(ctx, test.expected.Auth.TeamID)
-			require.NoError(t, err)
-			require.Equal(t, test.expected, actualSelected)
-		} else if assert.Error(t, err) {
-			assert.Equal(t, test.err.Code, err.(*slackerror.Error).Code, test.desc)
-			require.Equal(t, test.expected, actualSelected)
-		}
-	}
-}
-
-func TestPrompt_TeamAppSelectPrompt_AppSelectPrompt_EnterpriseWorkspaceApps_HasWorkspaceAuth(t *testing.T) {
```

Comment:
🗣️ Tests that resolve authentications for enterprise workspace apps are found here:

https://github.com/slackapi/slack-cli/blob/f9b74e0086fcb202435e92edf8f572d357037e89/internal/prompts/app_select_test.go#L476-L545

Reply:
🗣️ This particular case might also not need to have a match now since authentication is performed on the enterprise, but please call me out if this doesn't seem right!

Reply:
I think it's better to keep this test for now, but we're at that point where we can consider removing the migration logic for folks who have auth'd against an Enterprise Workspace and have their auth migrated to the Organization. At this point, I imagine most of those tokens have expired anyhow.